near-sdk/src/test_utils/test_env.rs

```diff
@@ -32,11 +33,13 @@ pub fn setup_with_config(vm_config: VMConfig) {
     )));
 }
 
+/// Setup the blockchain interface with a default configuration.
 pub fn setup() {
     setup_with_config(VMConfig::default());
 }
 
 // free == effectively unlimited gas
+/// Sets up the blockchain interface with a [`VMConfig`] which sets the gas costs to zero.
 pub fn setup_free() {
```

Comment:
Didn't want to add a comment on this, but this doesn't update the runtime configuration, which also contains gas costs. I still haven't resolved when each is used or why they are separate, can someone shed some light?